### PR TITLE
Add more 3rd party plugins to docs

### PR DESCRIPTION
**Pull request by maauer**

### CODE REVIEW

File: general/server/plugins/index.md [hunk 1]
```diff
@@ -271,6 +271,46 @@ Takes your tuners in TVHeadEnd and emulates a HDHomeRun, in order to connect to
...
+
+* [GitHub](https://github.com/jesseward/jellyfin-plugin-lastfm)
+
+#### VGMdb
+
+Adds support for VGMdb to music libraries. Can provide both images and metadata for artists and albums.
+
+**Links:**
+
+* [GitHub](https://github.com/nielsvanvelzen/jellyfin-plugin-vgmdb)
```
Comment (nielsvanvelzen) [inline general/server/plugins/index.md:312]: Hey I know this one!

**Review by nielsvanvelzen — APPROVED** [review]

File: general/server/plugins/index.md [hunk 2]
```diff
@@ -271,6 +271,46 @@ Takes your tuners in TVHeadEnd and emulates a HDHomeRun, in order to connect to
...
+
+Download flashy intros from prerolls.video for your movies.
+
+**Links:**
+
+* [GitHub](https://github.com/dkanada/jellyfin-plugin-intros)
+
+#### Youtube Metadata
+
+Download metadata about Youtube videos using a Youtube API key.
```
Comment (Maxr1998) [inline general/server/plugins/index.md:292]:
```suggestion
Downloads metadata of YouTube videos with a YouTube API key.
```

File: general/server/plugins/index.md [hunk 3]
```diff
@@ -271,6 +271,46 @@ Takes your tuners in TVHeadEnd and emulates a HDHomeRun, in order to connect to
...
+
+#### Intros
+
+Download flashy intros from prerolls.video for your movies.
+
+**Links:**
+
+* [GitHub](https://github.com/dkanada/jellyfin-plugin-intros)
+
+#### Youtube Metadata
```
Comment (Maxr1998) [inline general/server/plugins/index.md:290]:
```suggestion
#### YouTube Metadata
```

**Review by Maxr1998 — CHANGES_REQUESTED** [review]

File: general/server/plugins/index.md (hunk 3 above)
Comment (Maxr1998) [reply general/server/plugins/index.md:290]: I think you missed this one :smile:

Comment (maauer) [reply general/server/plugins/index.md]: Oops. Im still figuring out how pull requests work.

Comment (Maxr1998) [reply general/server/plugins/index.md]: No worries, thanks for the fix!

**Review by Maxr1998 — APPROVED** [review]

**Pull request merged**